client/post-editor/media-modal/secondary-actions.jsx

```diff
@@ -73,8 +54,9 @@ const MediaModalSecondaryActions = React.createClass( {
 		if ( ModalViews.LIST === view && selectedItems.length ) {
 			buttons.push( {
 				key: 'edit',
-				value: this.translate( 'Edit' ),
+				text: this.translate( 'Edit' ),
```

Comment:
In the media modal view, the primary action on top is edit, while the bottom primary action is continue. @artpi @iamtakashi  should we try to match this?